### ISSUE TITLE
[a11y]: TabPanel without interactive elements not scrollable

### ISSUE THREAD

**Issue by miriam-g** [carbon-design-system/carbon#19077]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.57.0

### React version

18.2.0

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker

### Description

We have a tearsheet with different tabs and tabpanels. One panel has non-interactive content that is that large that it has a scrollbar. Scrolling doesn't work with the keyboard.

https://github.com/user-attachments/assets/2f16fc55-cd03-4197-800a-be6fd526e6c9

I see this violation when using the Accessibility Checker:

![Image](https://github.com/user-attachments/assets/9c9905ee-7974-4046-906e-9d5daec0ac70)

### Reproduction/example

[carbon tearsheet with tabs](https://carbon-for-ibm-products.netlify.app/iframe.html?globals=viewport%3Abasic&id=ibm-products-components-tearsheet--with-navigation&viewMode=story)

### Steps to reproduce

To reproduce the issue, use this link: [carbon tearsheet with tabs](https://carbon-for-ibm-products.netlify.app/iframe.html?globals=viewport%3Abasic&id=ibm-products-components-tearsheet--with-navigation&viewMode=story) change the content on one tab that it doesn't fit on the page and use the keyboard to navigate through it.

### Suggested Severity

Severity 2 = User cannot complete task, and/or no workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by 2nikhiltom** [comment]
[Opened in ibm-products](https://github.com/carbon-design-system/ibm-products/issues/7521) -> fixed by https://github.com/carbon-design-system/ibm-products/pull/7536
